README.md

```diff
@@ -1,32 +1,60 @@
 # Superformula Mobile Developer Coding Test
 
-Make sure you read **all** of this document carefully, and follow the guidelines in it.
+## Dependencies
 
-## Requirements
+- bloc
+- dio
+- flutter_bloc
+- freezed
+- get_it
+- intl
+- json_serializable
+- qr_code_scanner
+- qr_flutter
+- bloc_test
+_ build_runner
+_ mocktail
```

Comment:
suggestion:  I❤️ ❤️❤️  the packages you decided to use for this....  I think some basic information on what they are and how to use them would be helpful though.  If someone had never used `freezed` for example, they would potentially have a difficult time understanding why it's so awesome and why they should keep using it.  :)